plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/controller-roles.yaml

```diff
@@ -925,6 +931,18 @@ items:
     - create
     - patch
     - update
+  - apiGroups:
+    - ""
+    resources:
+    - pods
+    verbs:
+    - get
+  - apiGroups:
+    - ""
+    resources:
+    - pods/status
```

Comment:
is there a way to group this with the existing update pods/status permission above? it makes it way easier to see this is not actually expanding the power of this controller

Reply:
There are two ways I guess:
1. add the "patch" permission to pods/status unconditionally since the "update" is already there
2. process the permissions in the test to group together the verbs based on equality of other fields in the `PolicyRule` structure.

I pushed a commit implementing 1. which seems pragmatic. I do not like 2 as it might not be trivial
and may require maintenance if new fields are added to the `PolicyRule` structure

Reply:
that sounds good to me, thanks